doc/article/en-US/dialog-basics.md

```diff
@@ -237,10 +241,28 @@ You can specify global settings as well for all dialogs to use when installing t
   </source-code>
 </code-listing>
 
-### Controller Settings
+### Dialog Settings
 
 The settings available for the dialog are set on the dialog controller on a per-dialog basis.
 - `viewModel` can be url, class reference or instance.
+  - url - path relative to the application root.
+    ```
+    .
+    +-- app.js
+    +-- forms
+        +-- consent-form.js
+        +-- consent-form.html
+    +-- prompts
+        +-- prompt.js
+        +-- prompt.html
+    ```
+    If you want to open a `prompt` from `consent-form` the path will be `prompts/prompt`.
+    > Warning: WebPack users should use `PLATFORM.moduleName` when passing a path. Not using it may lead to the resource not being bundled, when not referenced anywhere else.
+  - object - it will be used as the view model. In this case `view` must also be specified.
+  - class - the view model *class* or *constructor function*.
+    > Note: This approach depends on being able resolve the origin of the class.
+
+    > Warning: Known WebPack issues when using *DllPlugin*, *ModuleConcatenationPlugin*.
```

Comment:
I would rephrase: "this approach might not work when using `DLLPlugin` or `ModuleConcatenationPlugin`."
And add a note to the effect of:
"Webpack users are advised to always use the `string` overload wrapped with `moduleName`"